app/src/main/java/net/gsantner/markor/activity/DocumentActivity.java

```diff
@@ -85,7 +82,7 @@ public static void launch(Activity activity, File path, Boolean isFolder, Boolea
             intent.addFlags(Intent.FLAG_ACTIVITY_SINGLE_TOP);
         }
         nextLaunchTransparentBg = (activity instanceof MainActivity);
-        new ActivityUtils(activity).animateToActivity(intent, false, 0).freeContextRef();
+        new ActivityUtils(activity).animateToActivity(intent, false, null).freeContextRef();
```

Comment:
this one alone is the window fix right? (0=finish activity)

Reply:
Yes